.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- Welcome! Thanks for your issue. In order for us to investigate your issue, it really helps to make sure we have the information we need. Please fill out the form sections below so we can help you as promptly as we can. Thanks! -->
+
+## Tell us about your runtime:
+
+* **OS**:
+* **QUnit version**:
+* **How are you running QUnit? (e.g., browser, Grunt)**:
```

Comment:
I think this may potentially conflate two separate questions/pieces of information:
1. What environment are you running QUnit in? (e.g., browser, node)
2. How are you running QUnit? (e.g., testem, grunt, script, etc.)